docs/docs/self-hosting/troubleshooting/uploads.md

```diff
@@ -1,13 +1,54 @@
 ---
-title: Uploads failing
+title: Uploads
 description: Fixing upload errors when trying to self host Ente
 ---
 
-# Uploads failing
+# FAQ's regarding S3
```

Comment:
This section is not specific to S3 right? "Troubleshooting upload failures" might be a better fit?